astro/src/pages/privacy-policy.astro

```diff
@@ -5,5 +5,115 @@ import ThemedSection from '../components/ThemedSection.astro';
 
 <Layout title="Privacy Policy">
  <ThemedSection style="secondary">
+  <p>Last updated: <time datetime="2023-01-15">15 January 2023</time></p>
```

Comment:
It's 2024. :)
Perhaps right-align this?

Reply:
@brian-montgomery, that was the date on the document I got, so I just copied it over.